src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Providers/GetAzureProviderCmdlet.cs

```diff
@@ -89,5 +91,38 @@ public override void ExecuteCmdlet()
                 this.WriteObject(providers, enumerateCollection: true);
             }
         }
+
+        private PSResourceProvider[] ListPSResourceProviders()
+        {
+            var providers = this.ResourceManagerSdkClient.ListResourceProviders(
+                providerName: this.ProviderNamespace, 
+                listAvailable: this.ListAvailable);
+
+            if (string.IsNullOrEmpty(this.Location))
+            {
+                return providers
+                    .Select(provider => provider.ToPSResourceProvider())
```

Comment:
not sure this method is introduced in this CR, but good to spell it out here too